File: Android/Martinshare/app/src/main/java/mobile/martinshare/com/martinshare/API/protocols/GetActivityProtocol.java
package mobile.martinshare.com.martinshare.API.protocols;

/**
 * Created by Modestas Valauskas on 03.06.2015.
 */
public interface GetActivityProtocol {
    public void startedGetting();
    public void notLoggedIn();
    public void noInternet();
    public void unknownError();
    public void aktualisiert(boolean warn, String fetchedData);
}

File: iOS/Martinshare/Martinshare/GetActivityProtocol.swift
//
//  GetActivityProtocol.swift
//  Martinshare
//
//  Created by Modestas Valauskas on 05.02.16.
//  Copyright © 2016 ModestasV Studios. All rights reserved.
//

import Foundation

protocol GetActivityProtocol {
    
    func startedGetting()
    func notLoggedIn()
    func unknownError(_ string: String)
    func noInternet()
    func aktualisiert(_ array: NSArray, warn: Bool, afterAktualisiertOderAktuell: (()-> Void))
}
